todo/items/routes.py

```diff
@@ -0,0 +1,59 @@
+from typing import List
+
+from fastapi import APIRouter
+from fastapi import Depends
+from fastapi import HTTPException
+from fastapi import status
+from bson import ObjectId
+
+from ..users.routes import fastapi_users
+from ..users.models import User, UserModel
+from ..db import engine
+from .models import Item
+
+router = APIRouter(
+    prefix='/items',
+    tags=['items']
+)
+
+
+async def get_active_user(
+        user: User = Depends(fastapi_users.current_user(active=True))):
```

Comment:
FastAPI is starting to feel like Wordpress to me 😅

Reply:
I haven't done wordpress yet 😅